pkgs/applications/video/ani-cli/default.nix

```diff
@@ -22,14 +30,20 @@ stdenvNoCC.mkDerivation rec {
   };
 
   nativeBuildInputs = [ makeWrapper ];
+  runtimeDependencies = [ gnugrep gnused curl fzf ffmpeg aria2 ]
+    ++ lib.optional (player == "mpv") mpv
+    ++ lib.optional (player == "vlc") vlc
+    ++ lib.optional (player == "iina") iina
```

Comment:
Probably an assertion before `stdenvNoCC.mkDerivation` that `player` is one of the three.